specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -1235,6 +1334,32 @@
             "type": "string",
             "description": "name of the Compliance",
             "x-ms-parameter-location": "method"
+        },
+        "ResourceId": {
+          "name": "resourceId",
+          "in": "path",
+          "required": true,
+          "type": "string",
+          "description": "The identifier of the resource.",
+          "x-ms-parameter-location": "method"
+        },
+        "AdvancedThreatProtectionSettingName": {

```

Comment:
if current is the only value supported for the setting name we should make an enum with one value.

Reply:
Also do you support a list API?

Reply:
I changed it to enum with one value, thanks for the comment.

Reply:
We don't support LIST in our API, the only available type of settings name right now is "current".